ui/package.json

```diff
@@ -8,6 +8,7 @@
     "@testing-library/user-event": "^7.1.2",
     "antd": "^4.6.5",
     "axios": "^0.20.0",
+    "d3": "^6.2.0",
```

Comment:
I assume you wanted to install `chart.js` here, you should run `npm install chart.js` in the `ui/` folder rather than root. It'd
also be useful to remove the root package.json file:
```suggestion
```